feature-libs/asm/components/csagent-login-form/csagent-login-form.component.html

```diff
@@ -28,6 +41,9 @@
       cxPasswordVisibilitySwitch
     />
     <cx-form-errors
+      [translationParams]="{
+        label: 'asm.loginForm.password.label' | cxTranslate
+      }"
       [control]="csAgentLoginForm.get('password')"
     ></cx-form-errors>
```

Comment:
[MEDIUM]
Doesn't this case need `*cxFeature=ourFlag` + a counterpart `<cx-form-errors *cxFeature=...`> with `!ourFlag`? 0 as in all other examples?

Reply:
Thanks, found additional occurrences. Changed.